netlify.toml

```diff
@@ -596,3 +596,48 @@
 [[redirects]]
     from = "/signup"
     to = "/pricing"
+
+# Added: 2021-08-02
+[[redirects]]
+    from = "/docs/api/index"
+    to = "/docs/api/overview"
+
+# Added: 2021-08-02
+[[redirects]]
+    from = "/docs/contribute/index"
+    to = "/docs/contribute/overview"
+
+# Added: 2021-08-02
+[[redirects]]
+    from = "/docs/integrate/index"
+    to = "/docs/integrate/overview"
+
+# Added: 2021-08-02
+[[redirects]]
+    from = "/docs/plugins/build/index"
+    to = "/docs/plugins/build/overview"
+
+# Added: 2021-08-02
+[[redirects]]
+    from = "/docs/plugins/index"
+    to = "/docs/plugins/overview"
+
+# Added: 2021-08-02
+[[redirects]]
+    from = "/docs/self-host/index"
+    to = "/docs/self-host/overview"
+
+# Added: 2021-08-02
+[[redirects]]
+    from = "/docs/tutorials/event-tracking-guide"
+    to = "/docs/tutorials/actions"
+
+# Added: 2021-08-02
+[[redirects]]
+    from = "/docs/tutorials/index"
+    to = "/docs/tutorials/overview"
+
+# Added: 2021-08-02
+[[redirects]]
+    from = "/docs/user-guides/index"
+    to = "/docs/user-guides/overview"
```

Comment:
These seem incorrectly added. A redirect `log-in-with-sso` → `sso` seems missing on the other hand.

Reply:
`/overview` no longer exist. I'd recommend that this is rebased from `master` to ensure the changes in https://github.com/PostHog/posthog.com/pull/1681 are picked up. If that's already happened then we'll need to investigate what `safe_redirects.py` is doing.

Reply:
Great catch @Twixes, all set!